src/i18n/es/index.js

```diff
@@ -0,0 +1,156 @@
+import potato from './potato';
+import print from './print';
+import tester from './tester';
+export default {
+  en: {
+    message: {
+      ...potato,
+      help: {
+        label: 'Abrir tutorial'
+      },
+      print: { ...print },
+      tester: { ...tester },
+      keyboard: {
+        label: 'teclado'
+      },
+      layout: {
+        label: 'plano'
+      },
+      keymapName: {
+        label: 'Nombre de keymap',
+        placeholder: 'nombre de keymap personalizado'
+      },
+      loadDefault: {
+        label: 'Cargar valor por defecto',
```

Comment:
can we use something a bit shorter please?
![Screen Shot 2019-10-03 at 09 08 39](https://user-images.githubusercontent.com/2275667/66226275-4bec4d00-e68f-11e9-94ff-425caab168e7.png)

Reply:
I've set it to "Load Original"as there's really no short word for default in Spanish

Reply:
Will  `Cargar predeterminado` be short enough?